sopel/irc/__init__.py

```diff
@@ -301,6 +295,17 @@ def on_error(self):
         self.last_error_timestamp = datetime.now()
         self.error_count = self.error_count + 1
 
+    def change_current_nick(self, new_nick):
+        """Change the current nick without configuration modification.
+
+        :param str new_nick: New nick to be used by the bot.
```

Comment:
This doesn't follow our normal style for `:param:` docs.

```suggestion
        :param str new_nick: new nick to be used by the bot
```